frontend/src/hooks/useTrip.ts

```diff
@@ -53,11 +53,13 @@ export default function useTrip(id: string) {
     const removeLocationFromTrip = (deletedLocation: Location) => {
         const updatedTrip = {...trip, locations: trip.locations.filter(location => location.id !== deletedLocation.id)}
         setTrip(updatedTrip);
+        updateTripQuery(updatedTrip);
```

Comment:
ich glaube das `setTrip` brauchen wir hier nicht, das passiert ja in der `updateTripQuery` auch schon

Reply:
Ich bin mir auch nicht ganz sicher warum, aber wenn das updateTripQuery nicht mit drin ist, speichert es die gelöschten locations nicht korrekt. Die Gelöschten werden zwar nicht mehr angezeigt, kommen aber wieder wenn man auf Stray klickt